scripts/hca-utils/hca_utils/main.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import argparse
+
+from hca_utils import __version__ as hca_utils_version
+from .utils import HcaUtils
+
+
+class DefaultHelpParser(argparse.ArgumentParser):
+    def error(self, message):
+        """Print help message by default."""
+        sys.stderr.write(f'error: {message}\n')
+        self.print_help()
+        sys.exit(2)
+
+
+def run(arguments=None):
+    parser = DefaultHelpParser(description='A simple HcaUtils CLI.')
+    parser.add_argument('-V', '--version', action='version', version='%(prog)s ' + hca_utils_version)
+    parser.add_argument('-e', '--env', help='The Jade environment to target, defaults to dev', choices=['dev', 'prod'],
+                        default='dev')
+    parser.add_argument('-p', '--project', help="The Jade project to target, defaults to correct project for dev")
+    parser.add_argument('-d', '--dataset', help="The Jade dataset to target")
+    parser.add_argument('-t', '--task', help="Just check for problems or perform a soft delete to address the problems",
```

Comment:
semantically i usually prefer 'do a thing' vs 'just check to see what needs done' to be defined by the presence/absence of a flag, rather than choosing between two mandatory values.